app/views/user/dashboard/index.html.erb

```diff
@@ -1,11 +1,3 @@
-<% if @requests.any? %>
```

Comment:
Check with @rosspatel01 , but I think he meant that this section should be after the heading, rather than after all of the documents

Reply:
Good point, I might have interpreted it differently :stuck_out_tongue: